alarm/models.py

```diff
@@ -66,4 +84,16 @@ def refresh_candle_data(cls, trade_pair, high_price, low_price):
         return candle
 
     def __str__(self):
-        return f"{self.trade_pair}: {self.low_price}-{self.high_price}"
+        return f"{self.low_price}-{self.high_price}"
+
+
+class ThresholdBrake(models.Model):
+    threshold = models.ForeignKey(Threshold, on_delete=models.CASCADE)
+    happened_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"{self.threshold}"
+
+    def delete_old_threshold_brakes(self):
```

Comment:
Ok, I want to delete threshold brakes for a specific threshold. How will I do that for a terminal given a threshold. Filter brakes by it and that call this function. Instead of just threshold.delete_old_threshold_brakes. Please think about it an explain why it is better to place it to threshold. Make a note, how you operate here on a set of threshold brakes objects vs a single object data.

Reply:
I deleted some of this code, it will be in another branch that relates specifically to the call.